membership-attribute-service/test/repositories/MembershipAttributesSerializer.scala

```diff
@@ -1,42 +0,0 @@
-package repositories
-
-import com.amazonaws.services.dynamodbv2.model.AttributeValue
-import com.github.dwhjames.awswrap.dynamodb._
-import models.Attributes
-import org.joda.time.LocalDate
-
-object MembershipAttributesSerializer {
-  object AttributeNames {
-    val userId = "UserId"
-    val membershipNumber = "MembershipNumber"
-    val tier = "Tier"
-    val membershipJoinDate = "MembershipJoinDate"
-  }
-}
-
-case class MembershipAttributesSerializer(tableName: String)
-  extends DynamoDBSerializer[Attributes] {
```

Comment:
I'm not sure where this serializer was being used.. I removed it and all the tests still passed